src/tsday/dto/update-tsday.dto.ts

```diff
@@ -17,35 +17,38 @@
 import { ApiProperty } from '@nestjs/swagger';
 import { IsInt, Max, Min } from 'class-validator';
 
-export class UpdateTsdayDto {
+const MIN_MINS = 0;
+const MAX_MINS = 720;
```

Comment:
this is already defined in `create-tsday.dts.ts` - it might just need an `export` in front of it there, and not redefine it here